.github/workflows/integration.yml

```diff
@@ -12,6 +12,11 @@ jobs:
       matrix:
         python-version: [2.7, 3.9]
         downstream: [botocore, requests]
+      exclude:
+        # excludes botocore in Python 2.7
+        - python-version: 2.7
+          downstream: botocore
+
```

Comment:
This should be under `matrix`, the indentation is currently incorrect

Reply:
@pquentin Thanks, fixed it please have a look.